valhalla/baldr/datetime.h

```diff
@@ -169,9 +169,13 @@ bool is_restricted(const bool type,
  * @return Returns std::tm time structure. If the input string is not valid this method
  *         sets tm_year to 0.
  */
-static std::tm iso_to_tm(const std::string& iso) {
+static inline std::tm iso_to_tm(const std::string& iso) {
   // Create an invalid tm, then populate it from the ISO string using get_time
-  std::tm t = {0, -1, -1, -1, -1, 0, 0, 0};
```

Comment:
@TeXitoi what compiler are you using, does POD struct list initialization not work on your platform?

Reply:
GCC 6.3, I have a warning saying that not every fields are initialised.

Reply:
And glibc tm have a out of spec field.

Reply:
Thus, adding 2 more zeros remove the warning, but is not portable.

Reply:
Another solution that would fix my issue is to put this code in the `.cc`

Reply:
nah this is fine. we'll just have to note that this is a place where we cant use that due to some systems defining `tm` differently.